### PR TITLE
add homepage

### PR DESCRIPTION
**Pull request by Ayushi20-19**
image

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  top: 0;
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transition: all 1s;
+  z-index: 2;
+  /* background-color: var(--home-primary-color); */
```
Comment (NamrataRaikwar2002) [inline styles.css:76]: you can remove this comment.

**Review by NamrataRaikwar2002 — APPROVED** [review]
looks really good Ayushi. nice work.

File: styles.css (hunk 1 above)
Comment (Ayushi20-19) [reply styles.css:76]: yes i forget that

**Review by prakash855 — APPROVED** [review]
PR looks good!

**Pull request merged**